pkg/health/health.go

```diff
@@ -28,24 +30,33 @@ type Result struct {
 	Output  string
 }
 
-func (h ConsulHealthChecker) Service(serviceID string) (map[string][]Result, error) {
-	entries, _, err := h.client.Health().Service(serviceID, "", false, nil)
+func (h ConsulHealthChecker) Service(serviceID string) (map[string][][]Result, error) {
```

Comment:
We discussed this a bit in person, but we decided to consolidate this back to a `map[string]Result`. Yes, documenting this is valuable and encouraged.